randovania/data/json_data/prime1.txt

```diff
@@ -1580,411 +1583,799 @@ Tallon Overworld
 ----------------
 Landing Site
 Asset id: 2993688902
-> Dock to Gully; Heals? False
+> Dock to Gully; Heals? False; Spawn Point
   * Normal Door to Gully/Dock to Landing Site
+  > Save Station
+      Trivial
 
 > Dock to Canyon Cavern; Heals? False
   * Normal Door to Canyon Cavern/Dock to Landing Site
+  > Save Station
+      Trivial
 
 > Dock to Temple Hall; Heals? False
   * Normal Door to Temple Hall/Dock to Landing Site
+  > Save Station
+      Trivial
 
 > Dock to Alcove; Heals? False
   * Normal Door to Alcove/Dock to Landing Site
+  > Dock to Gully
+      Trivial
+  > Save Station
+      Trivial
 
 > Dock to Waterfall Cavern; Heals? False
   * Normal Door to Waterfall Cavern/Dock to Landing Site
+  > Save Station
+      Trivial
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 58
+  > Save Station
+      Trivial
+
+> Save Station; Heals? True
+  > Dock to Gully
+      Space Jump Boots
+  > Dock to Canyon Cavern
+      Trivial
+  > Dock to Temple Hall
+      Trivial
+  > Dock to Alcove
+      Space Jump Boots
+  > Dock to Waterfall Cavern
+      Trivial
+  > Pickup (Missile Expansion)
+      Morph Ball
```

Comment:
I would remove the path from Ship to Alcove, and have Gully connect to Alcove instead. Reason is that when tricks are added, you wouldn't have to add the same tricks to two nodes, because you'd go to the Gully door first.